packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,38 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [33.0.0]
+
+### Added
+
+- Added `Mutex` lock in the `updateNftMetadata` function ([#4325](https://github.com/MetaMask/core/pull/4325))
+- Token price API support for mantle network ([#4376](https://github.com/MetaMask/core/pull/4376))
+
+### Changed
+
+- **BREAKING:** `NftController` constructor argument `selectedAddress` has been removed. ([#4221](https://github.com/MetaMask/core/pull/4221))
+- **BREAKING:** `NftController` now requires `AccountsController:get{Account,SelectedAccount}` messenger actions. ([#4221](https://github.com/MetaMask/core/pull/4221))
+- **BREAKING:** `NftController` now requires `AccountsController:selectedEvmAccountChange` event. ([#4221](https://github.com/MetaMask/core/pull/4221))
+- **BREAKING:** `NftDetectionController` now requires `AccountsControlelr:getSelectedAccount` messenger actions. ([#4221](https://github.com/MetaMask/core/pull/4221))
+- **BREAKING:** `AccountTrackerController` now requires a controller messenger ([#4225](https://github.com/MetaMask/core/pull/4225))
+- **BREAKING:** Removed `getIdentities`, `getSelectedAddress` and `onPreferencesStateChange` from the constructor arguments of the `AccountTrackerController` ([#4225](https://github.com/MetaMask/core/pull/4225))
+- **BREAKING:** Bump dependency and peer dependency `@metamask/accounts-controller` to `^17.0.0` ([#4413](https://github.com/MetaMask/core/pull/4413))
+- Upgrade `TokenRatesController` to `BaseControllerV2` ([#4314](https://github.com/MetaMask/core/pull/4314))
+- Update Nft controllers to use `selectedAccountId` instead of `selectedAddress` ([#4221](https://github.com/MetaMask/core/pull/4221))
+- `TokenRatesController` uses checksum instead of lowercase format for token addresses ([#4377](https://github.com/MetaMask/core/pull/4377))
+- Bump `@metamask/keyring-api` to `^8.0.0` ([#4405](https://github.com/MetaMask/core/pull/4405))
+- Bump `@metamask/eth-snap-keyring` to `^4.3.1` ([#4405](https://github.com/MetaMask/core/pull/4405))
+- Bump `@metamask/keyring-controller` to `^17.1.0` ([#4413](https://github.com/MetaMask/core/pull/4413))
+
+### Removed
+
+- Remove `value` from price API responses ([#4364](https://github.com/MetaMask/core/pull/4364))
+
+### Fixed
+
+- Catch the case when data are not available for given token ([#4361](https://github.com/MetaMask/core/pull/4361))
+- Mutex released on the finally block of refresh function ([#4270](https://github.com/MetaMask/core/pull/4270))
```

Comment:
```suggestion
- Fix `refresh` method remaining locked indefinitely after it was run successfully. Now lock is released on successful as well as failed runs. ([#4270](https://github.com/MetaMask/core/pull/4270))
```

The changelog should convey user impact instead of implementation details.

Reply:
Sounds good to me, I just reported what was noted in the CHANGELOG section of that PR! :)

Reply:
@ccharly Yes no worries! We usually try to do another pass on the changelog entries while reviewing release candidates.